Change overworld database concept from review (#58)
Gamify-IT/docs: dev-manuals/services/overworld/database-model-and-type.md

## Player-specific data
- the player-specific data should also be stored in a SQL database, there are relations from unlocked areas to the defined areas, completed dungeons, tasks, ...

# Result

> SQL is the favorite here because of the many relations we have. For our project in the overworld, it does not make much difference whether we use MySQL or PostgreSQL, because the performance does not have many differences and the features of both databases are completely sufficient for us

Reviewer: ```suggestion
SQL is the favorite here because of the many relations we have. For our project, it does not make a difference what type of database system we use because the performance will be similar and the features of all databases are sufficient for us.
```

## Result after Review

> Of course, an orm is used. We have selected PostgreSQL as database.

Reviewer: ```suggestion
An ORM should be used. We have selected PostgreSQL as our database.
```
